server/interview/interview.controller.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const router = express.Router();
+
+const interviewService = require('./interview.service');
+
+//Create interview
+router.post('/', async function(req, res) {
+    const postBody = req.body;
+    const userId = postBody.userId;
+    const difficulty = postBody.difficulty;
+    const interviewId = postBody.interviewId;
+
+    if (interviewId) {
+        interviewService.updateInterview(userId, difficulty, interviewId)
+            .then(interview => {
+                res.json(interview);
+            });   
+    } else {
+        interviewService.createInterview(userId, difficulty)
+            .then(interview => {
+                res.json(interview);
+            }); 
```

Comment:
This API route does two many things... let's separate teh post and put routes into two separate routes

Reply:
Sure

Reply:
Separated it into a post and a put request